### ISSUE TITLE
NullPointerException in PrinterShare print method

### ISSUE THREAD

**Issue by ChesRomakhin** [hierynomus/smbj#383]
https://github.com/hierynomus/smbj/blob/ed244cf98a1f880ed9c638d95eb5b9a0ee41d1e5/src/main/java/com/hierynomus/smbj/share/PrinterShare.java#L51

Passing null as `path` parameter to openFileId causes NullPointerException on

https://github.com/hierynomus/smbj/blob/a11232c4f50158aa2ee1da1c284d71f973f51f03/src/main/java/com/hierynomus/mssmb2/messages/SMB2CreateRequest.java#L77

**Comment by hierynomus** [comment]
Thanks for finding this :) Fixed by passing the SmbPath from the Share.

**Comment by MathOliveira** [comment]
Hello,
I am having the same problem in my code. Am I doing something wrong?

```

        String user = "USER";
        String pass = "PASS";
        String domain = "DOMAIN";
        String share = "SHARE";
        String ip = "IP";

        SMBClient client = new SMBClient();
        Connection connection = client.connect(ip);
        AuthenticationContext ac = new AuthenticationContext(user, pass.toCharArray(), domain);
        Session session = connection.authenticate(ac);

        PrinterShare printer = (PrinterShare) session.connectShare(share);
        printer.print(new ByteArrayInputStream(commands.getBytes(StandardCharsets.UTF_8)));
        System.out.println(" Done ");
```

**Comment by hierynomus** [comment]
@MathOliveira Which version are you using?

**Comment by hierynomus** [comment]
I've just released 0.9.0 which should fix your issue.
